src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -206,7 +215,17 @@ export const Sidebar: React.FC<SidebarProps> = ({
                 <StartDate quoteCartId={quoteCartId} modal size="sm" />
               </Body>
               <Footer>
-                {isConnectPaymentAtSignEnabled ? (
+                {isLoadingQuoteCart ? (
+                  <Button
+                    size="sm"
+                    fullWidth
+                    style={{ height: '2.75rem' }}
+                    foreground={colorsV3.gray900}
+                    background={colorsV3.purple500}
+                  >
+                    <LoadingDots color={colorsV3.gray500} />
+                  </Button>
+                ) : isConnectPaymentAtSignEnabled ? (
```

Comment:
Wanna update this also so we just show the disabled state instead of multiple loading states?